README.md

```diff
@@ -70,52 +64,37 @@ IOmfIngressService omfIngressService = baseOmfIngressService.GetOmfIngressServic
 
 Note that the instance of the IOmfIngressService is scoped to a tenant and namespace. If you wish to work in a different tenant or namespace, you would need another instance scoped to that tenant and namespace.
 
-## OMF Connections
-
-An OMF Connection is made up of three components: one or more Clients, a Topic, and a Subscription. Data is sent to a Topic via a Client, where the data is buffered and made available for the Subscription. The Subscription relays data from the Topic to the Sequential Data Store in the namespace that the Subscription resides in. The OmfIngressClient in this example creates a connection using an existing clientId, and creating a Topic and Subscription as detailed below.
-
 ## Clients
 
-Devices sending OMF messages each need a clientId and clientSecret. The clientId is used route messages to the proper topic(s). ClientIds may be mapped to at most one topic per namespace. For more details on Clients see the [Client Credential Client Documentation](https://ocs-docs.osisoft.com/Content_Portal/Documentation/Identity/Identity_ClientCredentialClient.html).
+Devices sending OMF messages each need a clientId and clientSecret. The clientId is used route messages to the proper connection(s). ClientIds may be mapped to at most one connection per namespace. For more details on Clients see the [Client Credential Client Documentation](https://ocs-docs.osisoft.com/Content_Portal/Documentation/Identity/Identity_ClientCredentialClient.html).
 
-## Topics
+## OMF Connections
 
-A Topic is used to aggregate data received from clients and make it available for consumption via a Subscription. A topic must contain at least one client Id. Client Ids may be added to or removed from an existing topic. First, we create the Topic locally by instantiating a new Topic object:
+ An OMF Connection is the connection that handles reading and handling OMF messages. We create the Connection locally by instantiating a new OmfConnectionCreate object:
 
 ```C#
-Topic topic = new Topic()
+OmfConnectionCreate omfConnectionCreate = new OmfConnectionCreate()
 {
-    Name = "REPLACE_WITH_TOPIC_NAME",
-    Description = "This is a sample Topic",
+    Name = "REPLACE_WITH_CONNECTION_NAME",
+    Description = "This is a sample Connection",
     ClientIds = new List<string>() { mappedClientId }
```

Comment:
is this intentionally different than the code?  i know its effectively the same

Reply:
This was intentional. I wanted to scrub any reference to "Topic" since it is no longer used.